frontend/src/board/contextmenu/colors.tsx

```diff
@@ -32,3 +45,10 @@ export function colorsSubMenu({ board, focusedItems, dispatch }: SubmenuProps) {
         dispatch({ action: "item.update", boardId: b.id, items: updated })
     }
 }
+
+function itemColorOrDefault(items: Item[]) {
+    if (items.length === 0) return YELLOW
```

Comment:
May I suggest introducing a DEFAULT_NOTE_COLOR constant to be used here as well as in `newNote` and `cutCopyPasteHandler` at least?

Reply:
Added!